openfisca_us/variables/gov/states/pa/tax/income/pa_income_tax_before_forgiveness.py

```diff
@@ -0,0 +1,16 @@
+from openfisca_us.model_api import *
+
+
+class pa_income_tax_before_forgiveness(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "PA income tax before forgiveness"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.revenue.pa.gov/FormsandPublications/FormsforIndividuals/PIT/Documents/2021/2021_pa-40in.pdf#page=21"
+    defined_for = StateCode.PA
+
+    def formula(tax_unit, period, parameters):
+        taxable_income = tax_unit("pa_taxable_income", period)
+        rate = parameters(period).gov.states.pa.tax.income.rates
```

Comment:
This should resolve the test failure
```suggestion
        rate = parameters(period).gov.states.pa.tax.income.rate
```